src/lomc.jl

```diff
@@ -67,6 +67,19 @@ struct QMCState{
     replica::RRS
 end
 
+"""
+    _n_walkers(v, s_strat)
+Returns an estimate of the expected number of walkers as an integer.
+"""
+function _n_walkers(v, s_strat)
+    n = if hasfield(typeof(s_strat), :targetwalkers)
+        s_strat.targetwalkers
+    else # e.g. for LogUpdate()
+        walkernumber(v)
```

Comment:
Does this work if you start with a single walker and expect the computation to reach a very large number? Maybe we need to add a way for a `ShiftStrategy` to tell approximately how many walkers it will produce?

Reply:
`LogUpdate`, which does not have a field called `targetwalkers` damps any change in walkernumber. Thus you can expect the walkernumber to not be very different from the starting point. It doesn't work very well though. Then, if you do `DontUpdate` there is no way of knowing how high the walkernumber will grow. You'll have to set `maxlength` manually, anyway.
I'll try to clean up the `ShiftStrategy`s a bit more.